package.json

```diff
@@ -13,10 +13,11 @@
 		"node": ">=6"
 	},
 	"scripts": {
-		"test": "xo && ava"
+		"test": "xo && ava && tsd-check"
```

Comment:
Might be nice to run `tsd-check` before the tests. Just a suggestion.

Reply:
Hm, I don't have a preference about this. @sindresorhus what do you think?

Reply:
Running `tsd-check` is the least important of the 3 binaries for a JS project, and should be last like it is now.